packages/bruno-app/src/providers/ReduxStore/slices/collections/index.js

```diff
@@ -24,8 +23,9 @@ import {
 } from 'utils/collections';
 import { parseQueryParams, stringifyQueryParams } from 'utils/url';
 import { getSubdirectoriesFromRoot, getDirectoryName } from 'utils/common/platform';
+import os from 'os';
 
-const PATH_SEPARATOR = path.sep;
+const PATH_SEPARATOR = /Windows/i.test(os.release()) ? '\\' : '/';
```

Comment:
You can use `isWindows` function from `utils/common/platform`

Reply:
Done